package.json

```diff
@@ -56,7 +56,6 @@
     "@types/node": "^17.0.16",
     "@types/node-hid": "^1.3.1",
     "esbuild": "^0.17.4",
-    "parcel": "^2.9.3",
```

Comment:
there wasn't also a `source` field that was added?

Reply:
Yes, I didn't consider it necessary to remove it. But yes, let's remove it.